examples/keystatic/next.config.js

```diff
@@ -0,0 +1,4 @@
+/** @type {import('next').NextConfig} */
+module.exports = {
+  reactStrictMode: false,
```

Comment:
This should not be turned off - could you share more?

Reply:
```suggestion
```

Reply:
Keystatic used to require strict mode to be off, but this is no longer a thing!